app/src/main/java/com/example/drawingactivity/DrawViewModel.kt

```diff
@@ -213,8 +223,18 @@ class DrawViewModel : ViewModel() {
                 isDeleteDialogOpen,
                 isBottomSheetOpen,
                 isColorPickerOpen,
-                previousHSVColor
```

Comment:
See what I mean about trailing commas and git history?

Reply:
Yeah, looks problematic here since there isn't anything else.

I'll make sure to remember to use trailing commas from now on